src/applications/search/utils.js

```diff
@@ -12,3 +12,7 @@ export function truncateResponseString(string, maxLength) {
   }
   return `${string.slice(0, maxLength)}...`;
 }
+
+export function removeDoubleBars(string) {
+  return string.replace('| Veterans Affairs', '');
```

Comment:
Where does "| Veterans Affairs" come from?

Reply:
I believe it is coming from the metadata description from the CMS

Reply:
they are looking at fixing this on their end, but that may or may not happen, and if so they can't promise a timeline

Reply:
Oh I have write many a line of code for this exact reason haha.